### DISCUSSION TITLE
v0.12.0

### DISCUSSION THREAD

**Discussion by arturcic** [GitTools/actions#1049] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/GitTools/actions/releases/tag/v0.12.0'>v0.12.0</a>.</em>

**Comment by amyachin** [comment]
FYI:

After GitTools was updated to 0.12.0 on our **on-prem** Azure Devops Server I am getting the following error when trying to run a build pipeline with gitversion/* tasks: 

'##[error]No agent found in pool [xxxx]  which satisfies the following demand: Agent.Version. All demands: DotNetFramework, msbuild, visualstudio, vstest, Agent.Version -gtVersion **2.220.0**'  

in    #03651040cf31c565c173c98cc25b9f79454f0058 _minimumAgentVersion_ in task.json was bumped up from 2.114 to 2.220

Per our IT the devop server we use has no capability to automatically install/upgrade the agents, we are forced to manually update 2.220 from the current 2.181.

**Comment by arturcic** [comment]
Hi @amyachin that was on purpose, as we're preparing soon to have version 1.0.0 and we needed to pin the version of the agents our tasks work, and we decided to set v2.220.0 as its's the latest in the v2 series.

Starting with version 2.0.0 of our tasks we will target the v3 series of the agents.
